pkg/email/discussion_reply_created_originator.go

```diff
@@ -0,0 +1,76 @@
+package email
+
+import (
+	"html/template"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+)
+
+// DiscussionReplyCreatedOriginatorSubject is the subject for hte email
+type DiscussionReplyCreatedOriginatorSubject struct {
+	UserName string // The full name of the user replying
```

Comment:
Consider changing variable name to `FullName` or other similar convention so it's more self-descriptive

Reply:
I agree in principal, but I leaned on the side of naming consistency. Other fields that hold full name data in other email templates call this data UserName as well. My compromise for this was to add the comment to make it clear this is users full name.